Adding Changes of NFT Wipe Support to the temporary develop branch

gayanath8 opened the pull request.

palindaa committed:
Initial commit for serial number for retirement block

palindaa committed:
Change retirement block property to option

palindaa committed:
Revert the commit - Serial Number

palindaa committed:
Remove properties and move to variables

palindaa committed:
Update frontend config for UI update

palindaa committed:
Add meta data tags for serial number field

palindaa committed:
Fix validation issue in token id

palindaa committed:
persist data on docker compose down

palindaa committed:
Update validation in guadian service block as well

palindaa committed:
Enable wipe for NFT

palindaa committed:
Add log line to inspect the issue

palindaa committed:
Add new log line to wipe request

palindaa committed:
Add more logs to troubleshoot

palindaa committed:
add log line for permission

palindaa committed:
Add more logs

palindaa committed:
Update port mapping to access mongo db

palindaa committed:
Fix token bug issue

palindaa committed:
Add more log line to KEY TYPE ENTITY

palindaa committed:
Print the KEY_TYPE_KEY_ENTITY

palindaa committed:
Add more logs

palindaa committed:
Fix paramter pass issue

palindaa committed:
set amount or serial number based on the token type

palindaa committed:
Merge branch 'develop' into feature/wipe-serialno

palindaa committed:
Add log line to investigate user not found issue

palindaa committed:
Merge branch 'develop' into feature/wipe-serialno

palindaa committed:
Fix retirement block

Commit:
Merge remote-tracking branch 'upstream/develop' into wipe-serialno-latest

palindaa committed:
Fixes to serial number

palindaa committed:
Fix freezing issue

palindaa committed:
Fix .map bug

gayanath8 committed:
Wipe Block Changes:
- Added separate ui component for Wipe server block
- Added support to schema data input serial numbers for wipe

gayanath8 committed:
Code Cleanup:
- Removing unwanted logs from the codebase

gayanath8 committed:
Updating Documentation

palindaa commented on guardian-service/src/api/permission.service.ts:
```diff
@@ -312,13 +313,15 @@ export async function permissionAPI(logger: PinoLogger): Promise<void> {
+                            topicId: entityId
+                        })
                         return new MessageResponse(
-                            await dataBaseServer.count(Topic, {
```
@gayanath8  revert this change

gayanath8 committed:
Update Validation:
- Removed Validation for Wipe Block for property rule
- Added Validation policy-service for both token types

gayanath8 committed:
Code Cleanup-II
